docs/configuration/redis-configuration.md

```diff
@@ -1,4 +1,7 @@
 #Redis configuration
+
+Recently, RabbitMQ was replaced with Redis as a queue service and periodic task database. The reason for that were performance bottlenecks.
```

Comment:
The reason for that was to increase SC4SNMP performance and protect against bottlenecks.